mm/routines/product.py

```diff
@@ -15,3 +15,13 @@ def update_enum_product_attribute(self, sku, att_name, att_value):
                 sku, att_name, att_value)
 
         return record
+
+    def get_configurable_associated_products(self, store_id=None, is_active=None, is_visible=None):
+        rows = \
+            self.call(
+                'get_configurable_associated_products',
+                store_id,
+                int(is_active) if isinstance(is_active, bool) else None,
```

Comment:
I'm not sure if this makes sense. You're essentially only setting a value if it's a boolean type else ignoring. You might just *assert* that these are booleans or None at the top, that way someone doesn't pass in an integer and get unexpected results.